CONTRIBUTING.rst

```diff
@@ -60,6 +60,13 @@ Development Environment
      $ vagrant up
      $ vagrant ssh
 
+* If you prefer developing on Ubuntu, you will need libssl-dev and libffi-dev packages to run tests. If missing these packages, please install by running:
```

Comment:
We should probably have a style guide (starting with https://clusterhq.atlassian.net/browse/FLOC-1216?) because we have these half-rules which have built up but aren't documented and aren't always kept to. If you look at http://rhodesmill.org/brandon/2012/one-sentence-per-line/ as an inspiration it might be helpful - we tend to use one sentence per line for some of the reasons described there, such as easier diff-ing.